video_worker/api_communicate.py

```diff
@@ -52,9 +54,7 @@ def run_veda(self):
 
         self.veda_token = generate_apitoken.veda_tokengen()
         if self.veda_token is None:
-            ErrorObject().print_error(
-                message='VEDA API Conn Fail:\nInvalid Setup/Method'
-            )
```

Comment:
@yro Are we using these print statements anywhere for logs purpose OR is it safe to remove them ?